src/Winton.AspNetCore.Seo/Winton.AspNetCore.Seo.csproj

```diff
@@ -2,17 +2,21 @@
   
   <PropertyGroup>
     <Authors>Winton</Authors>
+    <Company>Winton</Company>
     <Copyright>Copyright 2017 Winton</Copyright>
     <Description>ASP.NET Core Library for adding SEO metadata to websites</Description>
     <GenerateDocumentationFile>True</GenerateDocumentationFile>
     <GetVersion>False</GetVersion>
     <NoWarn>$(NoWarn);SA1101;SA1309;SA1413</NoWarn>
     <PackageId>Winton.AspNetCore.Seo</PackageId>
-    <PackageRequireLicenseAcceptance>False</PackageRequireLicenseAcceptance>
+    <PackageIconUrl>https://raw.githubusercontent.com/wintoncode/Winton.AspNetCore.Seo/master/icon.jpg</PackageIconUrl>
+    <PackageLicenseUrl>https://raw.githubusercontent.com/wintoncode/Winton.AspNetCore.Seo/master/LICENSE</PackageLicenseUrl>
+    <PackageProjectUrl>https://github.com/wintoncode/Winton.AspNetCore.Seo</PackageProjectUrl>
+    <PackageRequireLicenseAcceptance>True</PackageRequireLicenseAcceptance>
```

Comment:
Does this cause one of those popups when installing the package?
I find them incredibly annoying and don't think they actually change/mean anything.

Reply:
Yes, that's what it does. I wasn't sure whether or not we'd need to have that turned on, so I aired on the side of caution. I don't have a strong opinion. User's can check a box in the VS nuget manager to tell it to not show those popups. 

Does anyone else have an opinion on whether or not we need to have this?